easybuild/easyconfigs/p/python-openstack/python-openstackclient-3.12.0-foss-2015b-Python-2.7.11.eb

```diff
@@ -0,0 +1,174 @@
+easyblock = 'Bundle'
+
+name = 'python-openstackclient'
+version = '3.12.0'
+
+homepage = "https://pypi.python.org/pypi/python-openstackclient/"
+description = """OpenStack Client"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+python = 'Python'
+pythonversion = '2.7.11'
+py_short_ver = '.'.join(pythonversion.split('.')[0:2])
```

Comment:
You're not actually using this `py_short_ver` variable at all, please include `Python` directly as dependency & use the `%(pyver)s` and ``%(pyshortver)s` templates:

```python
version = '3.12.0'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
    ...
]